src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -115,6 +115,30 @@ protected function redirectToRoute(string $route, array $parameters = [], int $s
         return $this->redirect($this->generateUrl($route, $parameters), $status);
     }
 
+    /**
+     * Returns a RedirectResponse to last accessed url.
+     *
+     * @param int    $status   HTTP Status code
+     * @param string $fallback Default redirect URL
+     *
+     * @final
+     */
+    protected function redirectBack(int $status = 302, string $fallback = '/'): RedirectResponse
+    {
+        // Get current request.
+        $request = $this->container
+            ->get('request_stack')
+            ->getCurrentRequest();
+
+        if (null === $request) {
+            throw new \LogicException('Request should exist so it can be processed for error.');
```

Comment:
"so it can be processed for error"? what does that mean?

Reply:
This message from `src/Symfony/Component/Security/Http/Authentication/AuthenticationUtils.php Line 78` :)